validators/register-validator-node.md

```diff
@@ -690,89 +710,99 @@ After the previous step delegation status will be seen as accepted.
 
--   `--gas-price` - Gas price value in Gwei for transaction (if not specified doubled average network value will be used)
+-   Certificate file (for example, fullchain.pem or cert.pem)
+-   Private key file (for example, privkey.pem, pk.pem)
 
+### Step 4.4: Upload certificates to the SKALE Node
 
-**Output:**
+Once you copied the certificate and private key file, all you have to do is to run the following command:
 
-> Node registered in SKALE manager. For more info run: skale node info
+```shell
+skale ssl upload -c $PATH_TO_CERT_FILE -k $PATH_TO_KEY_FILE
+```
 
-### Step 4.2: Check Node Status
+### SSL Status
 
-You can check the status of your node, and ensure that it is properly registered with the SKALE Network.
+Status of the SSL certificates on the node
 
 **Terminal Command:**
 
-```bash
-skale node info
-
+```shell
+skale ssl status
 ```
 
-**Output:**
-
-```bash
-# Node info
-Name: $NODE_NAME
-IP: $NODE_IP
-Public IP: <Public IP of Machine>
-Port: $NODE_PORT
-Status: Active
+Admin API URL: \[GET] `/api/ssl/status`
 
-```
 </Step>
 
 <Step id='five'>
 
-## 5: Upload new SSL certificates
+## 5: Register Node in SKALE Network
+
+### Step 5.1: Register Node with Node CLI
+
+Note: Before proceeding, you will need to have at least  **0.2 Test ETH**. Also amount of delegated skale tokens need to be more or equal to minimum staking amount. Otherwise you won't be able to register with the SKALE Testnet.
 
-> If you already have SSL certificate for your domain name, please add and manage this certificate in the SKALE node
+Note: Also, be sure the node has SSL certificates!
+
+To register with the network, you will need to provide the following:
+
+1.  Node name
+2.  Machine public IP
+3.  Port - beginning of the port range that will be used for skale schains (10000 recommended)
+4.  Domain name
 
 **Terminal Command:**
 
-```bash
-skale ssl upload
+```shell
+skale node register --name [NODE_NAME] --ip [NODE_IP] --port [PORT] --domain [DOMAIN_NAME]
+
 ```
 
-### Options
+Optional arguments:
 
--   `-c/--cert-path` - Path to the certificate file
--   `-k/--key-path` - Path to the key file
--   `-f/--force` - Overwrite existing certificates
+-   `--gas-price` - Gas price value in Gwei for transaction (if not specified doubled average network value will be used)
 
-Admin API URL: \[GET] `/api/ssl/upload`
+**Output:**
 
-### SSL Status
+> Node registered in SKALE manager. For more info run: skale node info
 
-Status of the SSL certificates on the node
+### Step 5.2: Check Node Status
+
+You can check the status of your node, and ensure that it's properly registered with the SKALE Network.
 
 **Terminal Command:**
 
-```bash
-skale ssl status
+```shell
+skale node info
 ```
 
-Admin API URL: \[GET] `/api/ssl/status`
+**Output:**
+
+```shell
+# Node info
+Name: $NODE_NAME
+IP: $NODE_IP
+Public IP: <Public IP of Machine>
+Port: $NODE_PORT
+Status: Active
```

Comment:
No domain output?

Reply:
@ebruoguzberk actually we've added domain name and node id to the `skale node info` cmd

Reply:
updated